hawkular/alerts.py

```diff
@@ -0,0 +1,119 @@
+import json
+import urllib2
+import urllib
+
+class HawkularAlertsError(urllib2.HTTPError):
+    pass
+
+class HawkularAlertsConnectionError(urllib2.URLError):
+    pass
+
+class HTTPErrorProcessor(urllib2.HTTPErrorProcessor):
+
+    def http_response(self, request, response):
+        if response.code < 300:
+            return response
+        return urllib2.HTTPErrorProcessor.http_response(self, request, response)
+
+    https_response = http_response
+
+class HawkularAlertsClient:
+    """
+    Creates a new client for Hawkular Alerting
+    """
+    def __init__(self,
+                 user,
+                 password,
+                 tenant_id,
+                 host='localhost',
+                 port=8080,
+                 path='hawkular/alerts'):
+
+        self.user = user
+        self.password = password
+        self.tenant_id = tenant_id
```

Comment:
In the python client, this parameter is called tenant_id.
I am not going to change it on this PR.